web/src/lib/components/shared-components/upload-panel.svelte

```diff
@@ -34,13 +34,12 @@
     in:fade={{ duration: 250 }}
     out:fade={{ duration: 250 }}
     on:outroend={() => {
-      notificationController.show({
-        message:
-          ($errorCounter > 0
-            ? `Upload completed with ${$errorCounter} error${$errorCounter > 1 ? 's' : ''}`
-            : 'Upload success') + ', refresh the page to see new upload assets.',
-        type: $errorCounter > 0 ? NotificationType.Warning : NotificationType.Info,
-      });
+      if ($successCounter > 0 && $successCounter !== $duplicateCounter) {
+        notificationController.show({
+          message: 'Upload success, refresh the page to see new upload assets.',
+          type: NotificationType.Info,
```

Comment:
With the duplicate count fixed, the `$successCounter !== $duplicateCounter` condition is no longer needed. You've also removed the notification that shows the error count. One possibility is restoring the original notification, but wrapping it in `if ($successCounter > 0 || $errorCounter > 0)` or you could rewrite it as:

```ts
if ($errorCounter > 0) {
  // Show error notification
} else if ($successCounter > 0) {
  // Show success notification
}
```

Reply:
Updated this in my latest commit.

Reply:
That looks good. Now instead of the comments you should actually show a notification with `notificationController.show({ ... });`